sdk/core/core-rest-pipeline/package.json

```diff
@@ -97,6 +97,7 @@
     "@azure/core-tracing": "^1.0.1",
     "@azure/core-util": "^1.11.0",
     "@azure/logger": "^1.0.0",
+    "@typespec/ts-http-runtime": "^0.2.1",
     "http-proxy-agent": "^7.0.0",
```

Comment:
Would we still need the proxy references if we're just exposing them from the ts-http-runtime?

Reply:
yeah we should be able to drop these, good catch

Reply:
Removed